src/openforms/authentication/models.py

```diff
@@ -102,3 +102,10 @@ class RegistratorInfo(BaseAuthInfo):
     class Meta:
         verbose_name = _("Registrator authentication details")
         verbose_name_plural = _("Registrator authentication details")
+
+        permissions = [
+            (
+                "can_register_client_submission",
```

Comment:
```suggestion
                "can_register_customer_submission",
```
perhaps that's a better term to avoid confusion with server/client, municipalities also always tend to speak about "klantreizen" so it lines up better.